package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@nasa-gcn/schema",
   "description": "Unified JSON Schema for multi-messenger and high-energy astronomy alerts",
-  "version": "4.1.0",
+  "version": "4.2.0",
```

```suggestion
  "version": "4.1.0",
```

Don't bump the version. We'll do that when we do a release.

---

Sorry, I think it results from one of the readme commands (maybe when I did npm `version minor`). I will try to correct it.

---

Hi, I have applied your suggestion and commit it. Is it enough to solve the problem or do you need me to do something else?
In my commit, this problem occurs twice, one in package.json and one in package-lock.json. If the problem is not solved, I saw that I can delete the files from my commit, would that solve the problem?